install-script/install.sh

```diff
@@ -1367,17 +1395,7 @@ function write_api_url() {
     local file
     readonly file="${2}"
 
-    local ext_indentation
-    readonly ext_indentation="${3}"
-
-    # ToDo: ensure we override only sumologic `api_base_url`
-    if grep "api_base_url" "${file}" > /dev/null; then
-        sed -i.bak -e "s/api_base_url:.*$/api_base_url: $(escape_sed "${api_url}")/" "${file}"
-    else
-        # write api_url on the top of sumologic: extension
-        sed -i.bak -e "1,/sumologic:/ s/sumologic:/sumologic:\\
-\\${ext_indentation}api_base_url: $(escape_sed "${api_url}")/" "${file}"
-    fi
+    write_yaml_config_key "${file}" "extensions sumologic api_base_url" "${api_url}"
```

Comment:
It'd be more idiomatic to separate the keys with `.` instead of ` `.

Reply:
I wouldn't say anything that's happening in this PR is idiomatic, but I'm happy to make this change :+1: